src/SourceBuild/content/eng/build.ps1

```diff
@@ -53,26 +53,24 @@ if ($help) {
 }
 
 $project = Join-Path $RepoRoot "build.proj"
-$arguments = @()
-$targets = "/t:Build"
+$actions = @("/p:Restore=true", "/p:Build=true", "/p:Publish=true")
 
-# This repo uses the VSTest integration instead of the Arcade Test target
 if ($test) {
   $project = Join-Path (Join-Path $RepoRoot "test") "tests.proj"
-  $targets += ";VSTest"
-  $arguments += "/p:Test=true"
+  $actions = @("/p:Restore=true", "/p:Build=true", "/p:Publish=true", "/p:IsTestRun=true")
 
   # Workaround for vstest hangs (https://github.com/microsoft/vstest/issues/5091) [TODO]
   $env:MSBUILDENSURESTDOUTFORTASKPROCESSES="1"
 }
 
+$arguments = @()
 # Override project if specified on cmd-line
 if ($projects) {
   $project = $projects
 }
 
 if ($sign) {
-  $arguments += "/p:Sign=true"
+  $arguments += "/p:DotNetBuildSign=true"
```

Comment:
For my understanding....you're using DotNetBuildSign=true here so that we don't end up trying to actually sign the orchestrator's 'artifacts'.

Reply:
Not only that. Arcade strips out a few properties before passing them down to the "solution build" step, including `Sign` and `Test`. That's why I had to use separate names (as the actual build step wasn't getting the properties I passed in).